src/features/embalm/stepContent/components/ArchaeologistListItem.tsx

```diff
@@ -0,0 +1,147 @@
+import { Flex, Image, Td, Text, Tr, Button, Checkbox, Spinner } from '@chakra-ui/react';
+import { Archaeologist } from '../../../../types/index';
+import { formatAddress } from 'lib/utils/helpers';
+import { selectArchaeologist, deselectArchaeologist } from 'store/embalm/actions';
+import { ethers } from 'ethers';
+import { Dispatch, SetStateAction, useState } from 'react';
+import { useAttemptDialArchaeologists } from '../../../../hooks/utils/useAttemptDialArchaeologists';
+import { useDialArchaeologists } from '../hooks/useCreateSarcophagus/useDialArchaeologists';
+
+import { useDispatch } from 'store/index';
+
+interface ArchaeologistListItemProps {
+  archaeologist: Archaeologist;
+  isSelected: boolean;
+  includeDialButton: boolean;
+  isDialing: boolean;
+  setIsDialing: Dispatch<SetStateAction<boolean>>;
+  onClick: () => void;
+}
+
+interface TableContentProps {
+  children: React.ReactNode;
+  icon: boolean;
+  checkbox: boolean;
+}
+
+export function ArchaeologistListItem({
+  isSelected,
+  archaeologist,
+  includeDialButton,
+  isDialing,
+  setIsDialing,
+  onClick,
+}: ArchaeologistListItemProps) {
+  const [isPinging, setIsPinging] = useState(false);
+  const { testDialArchaeologist } = useAttemptDialArchaeologists(setIsDialing);
+  const { pingArchaeologist } = useDialArchaeologists();
+  const dispatch = useDispatch();
+
+  const rowTextColor = isSelected ? (archaeologist.exception ? '' : 'brand.950') : '';
+
+  function TableContent({ children, icon, checkbox }: TableContentProps) {
+    return (
+      <Td isNumeric>
+        <Flex justify={icon || checkbox ? 'left' : 'center'}>
+          {icon && (
+            <Image
+              src="sarco-token-icon.png"
+              w="18px"
+              h="18px"
+            />
+          )}
+          {isPinging && checkbox ? (
```

Comment:
The `Spinner` causes the table row to jump around as `isPinging` changes. I vote that we remove the spinner and get with design on a better way to display the `isPinging` state.

Reply:
After discussing in design chat, we should just remove the spinner for now.